src/metabase/driver/generic_sql.clj

```diff
@@ -172,8 +172,8 @@
                                                (-fetch-page (inc page-num)))))))]
     (fetch-page 0)))
 
-(defn- table-rows-seq [_ database table-name]
-  (k/select (korma-entity database {:table-name table-name})))
+(defn- table-rows-seq [_ database table]
```

Comment:
The documentation for `table-rows-seq` says it's supposed to be passed a `^String` table name. Why not just add some preconditions to make sure `table-name` is non-empty a `String` and then fix the places that are calling the method incorrectly instead of changing the implementation to work around buggy code elsewhere and breaking it for people who call it correctly